Modules/EMCAL/src/RawCheck.cxx

```diff
@@ -5,7 +5,7 @@
 // This software is distributed under the terms of the GNU General Public
 // License v3 (GPL Version 3), copied verbatim in the file "COPYING".
 //
-// In applying this license CERN does not waive the privileges and immunities
+// In applyingthis license CERN does not waive the privileges and immunities
```

Comment:
This probably entered by accident. Needs to be fixed, I think this makes the PR checker fail.